charts/moneypenny/templates/ingress.yaml

```diff
@@ -32,7 +32,7 @@ spec:
       http:
         paths:
         {{- range .paths }}
-          - path: {{ . }}
+          - path: {{ . }}(/|$)(.*)
```

Comment:
I'm confused why this is necessary. There's no rewrite annotation that I can see so nothing uses `$1` and `$2`, and Moneypenny listens on `/moneypenny`, which matches the path defined in `values.yaml`.

Reply:
Argh, right, it's in the live annotation because I stole the template from nublado-proxy, and it looks like the ingress failed to sync.  Deleting and resyncing...

That seems to have been the issue.  PEBCAK.